src/status_im/ui/screens/chat/components/contact_request.cljs

```diff
@@ -0,0 +1,86 @@
+(ns status-im.ui.screens.chat.components.contact-request
+  (:require [quo.core :as quo]
+            [quo.react :as quo.react]
+            [quo.react-native :as rn]
+            [quo.design-system.colors :as quo.colors]
+            [status-im.i18n.i18n :as i18n]
+            [status-im.ethereum.stateofus :as stateofus]
+            [status-im.ui.screens.chat.components.style :as styles]
+            [re-frame.core :as re-frame]
+            [clojure.string :as string]))
+
+(def ^:private contact-request-symbol "↪ ")
```

Comment:
:private ?

Reply:
I guess because it's only used in the file, I believe the compiler will issue a warning if used outside (although it will allow usage)